js/src/get-started-page/faqs/index.js

```diff
@@ -105,47 +97,186 @@ const faqItems = [
+						'google-listings-and-ads'
+					) }
+				</p>
+				<p>
+					<ul>
+						<li>
+							{ __(
+								'General product information',
+								'google-listings-and-ads'
+							) }
+						</li>
+						<li>
+							{ __(
+								'Unique product identifiers',
+								'google-listings-and-ads'
+							) }
+						</li>
+						<li>
+							{ __(
+								'Data requirements for specific categories (auto-assigned by Google):',
+								'google-listings-and-ads'
+							) }
+							<ul>
+								<li>
+									{ __(
+										'Apparel & Accessories',
+										'google-listings-and-ads'
+									) }
+								</li>
+								<li>
+									{ __( 'Media', 'google-listings-and-ads' ) }
+								</li>
+								<li>
+									{ __( 'Books', 'google-listings-and-ads' ) }
+								</li>
+							</ul>
+						</li>
+					</ul>
+				</p>
+			</>
+		),
+	},
+	{
+		trackId:
+			'where-do-i-manage-my-product-feed-and-my-google-ads-campaigns',
 		question: __(
-			'What is a Performance Max campaign?',
+			'Where do I manage my product feed and my Google Ads campaigns?',
+			'google-listings-and-ads'
+		),
+		answer: (
+			<p>
+				{ __(
+					'You can manage and edit all of your products and your Google Ads campaigns right from your WooCommerce dashboard and on the WooCommerce Mobile App.',
+					'google-listings-and-ads'
+				) }
+			</p>
+		),
+	},
+	{
+		trackId: 'where-will-my-products-appear',
+		question: __(
+			'Where will my products appear?',
 			'google-listings-and-ads'
 		),
 		answer: (
 			<p>
 				{ createInterpolateElement(
 					__(
-						'Performance Max campaigns make it easy to connect your WooCommerce store to Google Shopping ads so you can showcase your products to shoppers across Google Search, Maps, Shopping, YouTube, Gmail, the Display Network and Discover feed to drive traffic and sales for your store. <link>Learn more.</link>',
+						'Once you start running a <linkPmax>Performance Max ads campaign</linkPmax>, your approved products will reach more shoppers to help grow your business by being shown on Google Search, Google Maps, the Shopping tab, Gmail, Youtube, the Google Display Network, and Discover feed.',
 						'google-listings-and-ads'
 					),
 					{
-						link: (
-							<AppDocumentationLink
-								context="faqs"
-								linkId="performance-max"
-								href="https://woocommerce.com/document/google-for-woocommerce/get-started/google-performance-max-campaigns"
-							/>
-						),
+						linkPmax,
 					}
 				) }
 			</p>
 		),
 	},
 	{
-		trackId: 'what-are-free-listings',
-		question: __( 'What are free listings?', 'google-listings-and-ads' ),
+		trackId: 'what-are-performance-max-campaigns',
```

Comment:
Since the tracking ID's are changing are we also going to run the script to regenerate the tracking documentation? Doesn't seem like that's a standard step of our release process so it might get missed if it's not included in the PR.